yarn.lock

```diff
@@ -3022,6 +3022,17 @@
   resolved "https://registry.yarnpkg.com/@hutson/parse-repository-url/-/parse-repository-url-3.0.2.tgz#98c23c950a3d9b6c8f0daed06da6c3af06981340"
   integrity sha512-H9XAx3hc0BQHY6l+IFSWHDySypcXsvsuLhgYLUGywmJ5pswRVQJUHpOsobnLYp2ZUaUlKiKDrgWWhosOwAEM8Q==
 
+"@hyperledger/cactus-api-client@1.1.2":
```

Comment:
@outSH This looks like it might be a leftover change, but I'm not sure. Please double check if setting up all the cross-package dependencies (one of our package depending on another one of our packages) to use `1.2.0` and then running `yarn install` makes this go away from the diff (or at least bumps up up to `1.2.0`

Reply:
@petermetz This entry was removed after `yarn install`

Reply:
> @petermetz This entry was removed after `yarn install`

@outSH Yaay, that's what I was hoping it would do in a best case scenario, thank you for the updates!